x-pack/test/api_integration/apis/secops/authentications.ts

```diff
@@ -5,11 +5,19 @@
  */
 
 import expect from 'expect.js';
+
 import { authenticationsQuery } from '../../../../plugins/secops/public/containers/authentications/index.gql_query';
 import { GetAuthenticationsQuery } from '../../../../plugins/secops/public/graphql/types';
-
 import { KbnTestProvider } from './types';
 
+const FROM = new Date('2000-01-01T00:00:00.000Z').valueOf();
+const TO = new Date('3000-01-01T00:00:00.000Z').valueOf();
+
+// typical values that have to change after an update from "scripts/es_archiver"
+const HOST_NAME = 'suricata-sensor-amsterdam';
+const TOTAL_COUNT = 3;
+const EDGE_LENGTH = 1;
+
```

Comment:
Nice! Thanks for hoisting these up in all the different integration tests. This'll make fixing them after schema changes quite a bit nicer :)